_opensearch/install/tar.md

```diff
@@ -7,144 +7,455 @@ nav_order: 50
+      reserved: true
+      backend_roles:
+      - "admin"
+      description: "Admin user"
    ```
 
-1. In a separate window, enable the Performance Analyzer plugin:
+### Apply changes
 
-   ```bash
-   curl -XPOST localhost:9200/_plugins/_performanceanalyzer/cluster/config -H 'Content-Type: application/json' -d '{"enabled": true}'
-   ```
+Now that TLS certificates are installed and demo users were removed or assigned new passwords, the last step is to apply the configuration changes. This last configuration step requires invoking `securityadmin.sh` while OpenSearch is running on the host.
 
-   If you receive the `curl: (52) Empty reply from server` error, you are likely protecting your cluster with the security plugin and you need to provide credentials. Modify the following command to use your username and password:
+1. Start OpenSearch. It must be running for `securityadmin.sh` to apply changes.
+   ```bash
+   # Change directories
+   cd /path/to/opensearch-{{site.opensearch_version}}/bin
 
+   # Run the service in the foreground
+   ./opensearch
+   ```
+1. Open a separate terminal session with the host and navigate to the directory containing `securityadmin.sh`.
    ```bash
-   curl -XPOST https://localhost:9200/_plugins/_performanceanalyzer/cluster/config -H 'Content-Type: application/json' -d '{"enabled": true}' -u 'admin:admin' -k
+   # Change to the correct directory
+   cd /path/to/opensearch-{{site.opensearch_version}}/plugins/opensearch-security/tools
    ```
-
-1. Finally, enable the Root Cause Analyzer (RCA) framework
-
+1. Invoke the script. See [Apply changes using securityadmin.sh]({{site.url}}{{site.baseurl}}/security-plugin/configuration/security-admin/) for definitions of the arguments you must pass.
    ```bash
-   curl -XPOST localhost:9200/_plugins/_performanceanalyzer/rca/cluster/config -H 'Content-Type: application/json' -d '{"enabled": true}'
+   # You can omit the environment variable if you declared this in your $PATH.
+   OPENSEARCH_JAVA_HOME=/path/to/opensearch-{{site.opensearch_version}}/jdk ./securityadmin.sh -cd /path/to/opensearch-{{site.opensearch_version}}/config/opensearch-security/ -cacert /path/to/opensearch-{{site.opensearch_version}}/config/root-ca.pem -cert /path/to/opensearch-{{site.opensearch_version}}/config/admin.pem -key /path/to/opensearch-{{site.opensearch_version}}/config/admin-key.pem -icl -nhnv
    ```
 
-   Similar to step 4, if you run into `curl: (52) Empty reply from server`, run the command below to enable RCA
+### Verify that the service is running
 
-   ```bash
-   curl -XPOST https://localhost:9200/_plugins/_performanceanalyzer/rca/cluster/config -H 'Content-Type: application/json' -d '{"enabled": true}' -u 'admin:admin' -k
-   ```
+OpenSearch is now running on your host with custom TLS certificates and a secure user for basic authentication. The last step is verifying that the host is reachable by other clients.
```

Comment:
Because we already said "...the last step is to apply the configuration changes", would it be better to rephrase the second sentence here?

Reply:
Took me a minute to understand what you were referencing. Let me think on it for a moment.

Reply:
Rephrased - let me know what you think.